SYCL/Reduction/reduction_reducer_op_eq.cpp

```diff
@@ -156,29 +155,28 @@ template <typename T, typename BinaryOperation, OperationEqual OpEq,
           bool IsFP = false>
 int testBoth(queue &Q, T Identity) {
   int Error = 0;
-  Error += test<T, true, BinaryOperation, OpEq, IsFP>(Q, Identity);
-  Error += test<T, false, BinaryOperation, OpEq, IsFP>(Q, Identity);
+  Error += test<T, BinaryOperation, OpEq, IsFP>(Q, Identity);
```

Comment:
testBoth() seems to be unused after the changes, right? If Yes, please remove it.